kmsauth/__init__.py

```diff
@@ -74,11 +75,20 @@ def __init__(
                 'kms',
                 region=self.region
             )
+        if extra_context is None:
+            self.extra_context = {}
+        else:
+            self.extra_context = extra_context
         self.TOKENS = lru.LRUCache(4096)
         self.KEY_METADATA = {}
-        self._validate_generator()
+        self._validate_validator()
```

Comment:
Not sure I understood the function name before, but this really doesn't make sense to me now.